packages/config/config/devices/0x0086/templates/aeotec_template.json

```diff
@@ -0,0 +1,1132 @@
+		"valueSize": 1,
+		"unit": "%",
+		"minValue": 0,
+		"maxValue": 100,
+		"defaultValue": 50,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"percent_threshold_clamp1": {
+		"$import": "#percent_threshold",
+		"label": "Wattage Percentage Threshold (Clamp 1)"
+	},
+	"percent_threshold_clamp2": {
+		"$import": "#percent_threshold",
+		"label": "Wattage Percentage Threshold (Clamp 2)"
+	},
+	"percent_threshold_clamp3": {
+		"$import": "#percent_threshold",
+		"label": "Wattage Percentage Threshold (Clamp 3)"
+	},
+	"percent_threshold_wholehem": {
+		"$import": "#percent_threshold",
+		"label": "Wattage Percentage Threshold (Whole HEM)"
+	},
+	"reset_parameters": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Normal Operation",
+				"value": 0
+			},
+			{
+				"label": "Reset",
+				"value": 1
+			}
+		]
+	},
+	"enable_notifications_basic_only": {
+		"label": "Send Notifications to Associated Devices (Group 1)",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Basic CC",
+				"value": 1
+			}
+		]
+	},
+	"enable_notifications": {
+		"label": "Send Notifications to Associated Devices (Group 1)",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Hail CC",
+				"value": 1
+			},
+			{
+				"label": "Basic report CC",
+				"value": 2
+			}
+		]
+	},
+	"enable_notifications_0-3": {
+		"label": "Send Notifications to Associated Devices (Group 1)",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Send nothing",
+				"value": 0
+			},
+			{
+				"label": "Send hail cc",
+				"value": 1
+			},
+			{
+				"label": "Send basic report cc",
+				"value": 2
+			},
+			{
+				"label": "Send hail cc when using action button to change the output load state",
+				"value": 3
+			}
+		]
+	},
+	"enable_notifications_0-4": {
+		"label": "Send Notifications to Associated Devices (Group 1)",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 4,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Send hail cc",
+				"value": 1
+			},
+			{
+				"label": "Send basic cc report",
+				"value": 2
+			},
+			{
+				"label": "Send multilevel switch report",
+				"value": 3
+			},
```

Comment:
We should use the correct CC names:
`Hail CC`,  `Basic CC Report`, `Multilevel Switch CC Report`
(or maybe leave out the `CC`, but everywhere)